CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ See [0Ver](https://0ver.org/).
 
 ### Features
 
+- Add picky exceptions to `future_safe` decorator like `safe` has.
+- Remove FutureResultE
```

Comment:
Please, don't remove it just yet. Let's first provide an alternative. Moreover, maintaining an alias is simple enough.